config/crd/bases/o2ims-hardwaremanagement.oran.openshift.io_hardwareprofiles.yaml

```diff
@@ -0,0 +1,178 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.15.0
+  name: hardwareprofiles.o2ims-hardwaremanagement.oran.openshift.io
+spec:
+  group: o2ims-hardwaremanagement.oran.openshift.io
```

Comment:
Can do this in a followup PR but we want to change the group to make it more generic